src/Benchmarks.ClientJob/Latency.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Benchmarks.ClientJob
+{
+    public class Latency
+    {
+        public double Average { get; set; }
+        public double P50 { get; set; }
+        public double P75 { get; set; }
+        public double P90 { get; set; }
+        public double P99 { get; set; }
```

Comment:
I know `wrk` is this terse. But, suggest expanding these names to something like `Within50thPercentile` &hellip; to avoid future confusion.